src/util.js

```diff
@@ -107,3 +107,80 @@ export function format_memory_and_limit(usage, limit) {
+                    if (!newContainersMeta) {
+                        console.log(newContainersMeta);
+                        resolve({newContainers: newContainers, newContainersStats: newContainersStats});
+                    }
+                    inspectRet = newContainersMeta.map((container) => {
+                        return new Promise((resolve, reject) => {
+                            varlinkCall(PODMAN, "io.podman.InspectContainer", {name: container.id})
+                                    .then(reply => {
+                                        resolve(JSON.parse(reply.container));
+                                    })
+                                    .catch(ex => reject(new Error("Failed to do InspectContainer call:", ex, JSON.stringify(ex))));
+                        });
+                    });
+                })
+                .then(function() {
+                    const containerCount = newContainersMeta.length;
+                    Promise.all(inspectRet)
+                            .then((inspectRet) => {
+                                let runEle = newContainersMeta.filter((ele) => {
+                                    return ele.status === "running";
+                                });
+                                inspectRet.map((inspectRet) => {
+                                    newContainers[inspectRet.ID] = inspectRet;
+                                });
+                                if (runEle.length === 0 && Object.keys(newContainers).length === containerCount) {
+                                    resolve({newContainers: newContainers, newContainersStats: newContainersStats});
+                                }
+                            })
+                            .catch(ex => console.error("Failed to do InspectContainer call:", ex, JSON.stringify(ex)));
+                })
+
+                .then(function() {
+                    containerStatsRet = newContainersMeta.filter((ele) => {
+                        return ele.status === "running";
+                    }).map((container) => {
+                        return new Promise((resolve, reject) => {
+                            varlinkCall(PODMAN, "io.podman.GetContainerStats", {name: container.id})
+                                    .then(reply => resolve({ctrId: container.id, ctrStats:reply.container}))
+
+                                    .catch(ex => {
+                                        console.error("Failed to do GetContainerStats call:", ex, JSON.stringify(ex));
+                                        reject(new Error("Failed to do GetContainerStats call:", ex, JSON.stringify(ex)));
+                                    });
+                        });
+                    });
+                    countRun = containerStatsRet.length;
+                })
+                .then(() => {
+                    Promise.all(containerStatsRet)
+                            .then((containerStatsRet) => {
```

Comment:
Please name this argument differently. Up to here, `containerStatsRet` is a list of promises, now it's a list of promise results.